src/main/java/org/ecommerce/validations/validators/OrderStatusValidator.java

```diff
@@ -0,0 +1,16 @@
+package org.ecommerce.validations.validators;
+
+import jakarta.validation.ConstraintValidator;
+import jakarta.validation.ConstraintValidatorContext;
+import org.ecommerce.enums.OrderStatus;
+import org.ecommerce.util.Enums;
+import org.ecommerce.validations.constraints.ValidOrderStatus;
+
+
+public class OrderStatusValidator implements ConstraintValidator<ValidOrderStatus, OrderStatus> {
```

Comment:
Missing test

Reply:
Here's the commit with the tests: https://github.com/algutierrez-griddynamics/e-commerce/commit/1c7c622da02d7ece8502d9665998de9a63928f75